src/HttpClient/Guzzle.php

```diff
@@ -75,14 +83,25 @@ public function post($url, $data)
     {
         $client = $this->getClient($url);
         try {
-            $response = $client->post($url, array(\GuzzleHttp\RequestOptions::JSON => $data));
+            $response = $client->post(
+                $url,
+                array($this->formUrlencoded ? \GuzzleHttp\RequestOptions::FORM_PARAMS : \GuzzleHttp\RequestOptions::JSON => $data)
+            );
```

Comment:
The inline ternary operator isn't so pleasant here. In line with my suggestion above, the ternary could be replaced with `$this->contentType`.

```suggestion
            $response = $client->post($url, array($this->contentType => $data));
```

Reply:
OK, updated